### ISSUE TITLE
Support importing in completion specs

### ISSUE THREAD

**Issue by Tyriar** [withfig/autocomplete#209]
VS Code has code (stable) and code-insiders, I tried to write a spec for insiders based on stable and this didn't seem to work I believe because of the `import`:

```ts
import { completionSpec as stableCompletionSpec } from './code';

export const completionSpec: Fig.Spec = {
  ...stableCompletionSpec,
  name: "code-insiders",
  description: "Visual Studio Code Insiders"
};
```

**Comment by mattschrage** [comment]
Hmm this is interesting. Currently, the canonical way to pull in an existing spec would be using the `loadSpec` property, which reads the other completion spec at runtime. 

```
export const completionSpec: Fig.Spec = {
  name: "code-insiders",
  description: "Visual Studio Code Insiders",
  loadSpec: "code"
};
```

I suppose the advantage of using `import` instead is that it is more idiomatic JS, and potentially good to do this at compile time for performande. @cstrnt does this make sense to implement?

**Comment by cstrnt** [comment]
I think using `import` would make this behavior much easier, as well as opening up stuff like importing `options`, `args` and `subcommand` from other specs. Let's look for a solution for that :)

**Comment by Monkeyank** [comment]
Closed with #388
